### ISSUE TITLE
ImageResponse is not a constructor

### ISSUE THREAD

**Issue by BenjaminSchmitt** [dialogflow/dialogflow-fulfillment-nodejs#11]
When I try to instantiate a new ImageResponse, nodejs raises this error: **ImageResponse is not a constructor.**

What I simply did:

```javascript
const {ImageResponse} = require('dialogflow-fulfillment');
...
function testHandler(agent){
    let imageResponse = new ImageResponse('https://example.com/placeholder.png');    // error
    imageResponse.setImage('https://assistant.google.com/static/images/molecule/Molecule-Formation-    stop.png');
    agent.add(imageResponse)
}
```

Am I doing something wrong ?

**Comment by positlabs** [comment]
I think you just need Image, not ImageResponse

**Comment by matthewayne** [comment]
positlabs is correct.  Where are you seeing code using `ImageResponse`?

**Comment by positlabs** [comment]
The names are changed when they are exported, for some reason. https://github.com/dialogflow/dialogflow-fulfillment-nodejs/blob/master/src/dialogflow-fulfillment.js#L490
```
module.exports.Text = TextResponse;
module.exports.Card = CardResponse;
module.exports.Image = ImageResponse;
module.exports.Suggestion = SuggestionsResponse;
module.exports.Payload = PayloadResponse;
```

I understand this makes things a little bit simpler in terms of usage, but the docs don't match.

<img width="539" alt="screen shot 2018-03-16 at 11 34 52 am" src="https://user-images.githubusercontent.com/1918844/37538580-4850e39e-290e-11e8-88d7-aa30832ebc93.png">

I think the exports should match the docs. 

```
module.exports.TextResponse = TextResponse;
module.exports.CardResponse = CardResponse;
module.exports.ImageResponse = ImageResponse;
module.exports.SuggestionResponse = SuggestionsResponse;
module.exports.PayloadResponse = PayloadResponse;
```

**Comment by matthewayne** [comment]
Agreed.  I'm inclined to use `Text` in lieu of `TextResponse` to keep things simple and less verbose, any reason to use `TextResponse` instead of `Text`?

**Comment by positlabs** [comment]
I'm good with that. I'm mostly concerned about docs not matching up

**Comment by positlabs** [comment]
@matthewayne I can submit a PR for this, if you want.

**Comment by BenjaminSchmitt** [comment]
Ok the problem is solved with ```new Image()```, thank you all. Let me know when I can close this issue.

**Comment by vmkrish** [comment]
It seems like the old docs were not deleted from https://github.com/dialogflow/dialogflow-fulfillment-nodejs/tree/master/docs which has led to the docs not matching up with the correct usage.

**Comment by koraysels** [comment]
thank youuuuuuuu i have a tendency to look stuff up in the Sourcecode and it was easy to mis the export Names being different then the ClassNames! :D
